packages/manager/apps/pci-rancher/src/pages/create/Create.page.tsx

```diff
@@ -47,11 +38,17 @@ export default function Create() {
         ranchersQueryKeyValue,
         (old: RancherService[]) => [...old, data.data],
       );
-      trackingPage(`${TRACKING_PATH}::${TrackingEvent.add}-success`);
+      trackPage({
+        pageType: PageType.dashboard,
```

Comment:
**Question**: Is this mandatory as `Create.page.tsx` is already tracked under router?